plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java

```diff
@@ -53,11 +53,11 @@
 public abstract class BaseElasticsearchConnectorTest
         extends BaseConnectorTest
 {
-    private final String image;
-    private ElasticsearchServer elasticsearch;
+    protected final String image;
+    protected ElasticsearchServer elasticsearch;
```

Comment:
I think `BaseElasticsearchConnectorTest` should not hold infrastructure at all and so it should not define `createQueryRunner()` method at all. Reusing infrastructure code via inheritance may make code complex, it is hard then to understand what happens when.

Instead all implementation of `BaseElasticsearchConnectorTest` should define `createQueryRunner` on their own by using well defined building blocks to limit the amount of copy paste.

Reply:
Applied. Moved building the server logic to the concrete classes.